Apps/WebClient/src/Server/Controllers/CommentController.cs

```diff
@@ -112,17 +110,17 @@ public IActionResult Update([FromBody] UserComment comment)
         /// Deletes a UserComment from the database.
         /// </summary>
         /// <returns>The deleted UserComment wrapped in a RequestResult.</returns>
+        /// <param name="hdid">The user hdid.</param>
         /// <param name="comment">The comment to be deleted.</param>
         /// <response code="200">The note was deleted.</response>
         /// <response code="401">The client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpDelete]
-        [Authorize(Policy = UserPolicy.UserOnly)]
-        public IActionResult Delete([FromBody] UserComment comment)
+        [Route("{hdid}")]
+        [Authorize(Policy = UserProfilePolicy.Write)]
```

Comment:
Do we have Delete policy instead of using Write policy for Delete?

Reply:
Write is correct since it's about modification, not necessarily with what